src/contact.ts

```diff
@@ -110,6 +112,11 @@ export class Contact implements Sayable {
       star:       !!rawObj.StarFriend,
       stranger:   !!rawObj.stranger, // assign by injectio.js
       avatar:     rawObj.HeadImgUrl,
+      /**
+       * @see 1. https://github.com/nodeWechat/wechat4u/blob/master/src/interface/contact.js#L65
+       * @see 2. https://github.com/Urinx/WeixinBot/blob/master/README.md
+       */
+      brand:      !!rawObj.UserName && !rawObj.UserName.startsWith('@@') && (rawObj.VerifyFlag & 8) !== 0,
```

Comment:
I thought that will only disable the following ONE line?

Could you please have a check about it, because the new version of your code is difficult to read...